packages/transmute-framework/src/event-store-factory/index.js

```diff
@@ -49,9 +49,13 @@ module.exports = class EventStoreFactory {
 
     this.version = pack.version;
 
-    this.web3 = new Web3(
-      new Web3.providers.HttpProvider(web3Config.providerUrl)
-    );
+    if (window.web3) {
```

Comment:
does this break node.js compatibility? We used to have some smoke tests for verifying this kind of thing.

Reply:
Adding this